drivers/mmc/core/slot-gpio.c

```diff
@@ -39,12 +43,40 @@ static int mmc_gpio_get_status(struct mmc_host *host)
 	return ret;
 }
 
+#ifdef CONFIG_MMC_BLOCK_DEFERRED_RESUME
+void mmc_cd_prepare_suspend(struct mmc_host *host, bool pending_detect)
+{
+	struct mmc_gpio *ctx = host->slot.handler_priv;
+
+	if (!ctx)
+		return;
+
+	ctx->suspended = true;
+	ctx->pending_detect = pending_detect;
+}
+EXPORT_SYMBOL(mmc_cd_prepare_suspend);
+#endif
+
```

Comment:
Useless break. 58-60 must be removed.

Reply:
Oh yes!
Sure I'll fix this when I get some time